Formula/libjson-rpc-cpp.rb

```diff
@@ -14,6 +14,12 @@ class LibjsonRpcCpp < Formula
     sha256 "d3e24865583d50b44a833b08fb90093a2db62bda4615e3c734ffe5cd2bda23ff" => :high_sierra
   end
 
+  # no active development in upstream repo
+  # new work goes to https://github.com/jsonrpcx/json-rpc-cxx
+  # open issue tracker, https://github.com/cinemast/libjson-rpc-cpp/issues/300
+  # already broken with building from source in master branch
+  disable!
```

Comment:
This sounds like we need to move sources or deprecate it, why would we disable it right away? (also a good reason to use the `because:` option)

Reply:
I could not figure out which way to go, I agree, it can be deprecated option (or whatever option to make the CI build happy :) )